sensorlib/ReadMe.md

```diff
@@ -0,0 +1,86 @@
+# CARLA Sensor Library
+
+## Overview
+
+The sensor library provides a proxy sensor for the CARLA simulator, exposing additional features.
+
+This new feature is provided to enable vulnerable road user detection using a LIDAR sensor in the context of a
+CARLA-driven simulation. The data is intended to feed into a CARMA V2X system for data fusion with other sensors and
+action by CARMA platform vehicles.
+
+The SimulatedSensor acts as a wrapper to the CARLA Semantic Lidar Sensor, capturing the data and performing
```

Comment:
Can we add information about why we are wrapping the CARLA Semantic Lidar?

Reply:
Sure how's this:

`The SimulatedSensor acts as a wrapper to the CARLA Semantic Lidar Sensor, capturing the data and performing
post-processing to generate a list of detected objects upon request. The primary reason for processing data from CARLA
is to ensure noise is applied to the output signal, and customization is possible for occlusion and any other procedures
in the future. The native CARLA sensor does not provide signal noise, misdetection simulation, or customization.

`

Reply:
yeah that is fine.